### PR TITLE
A new image

### PR DESCRIPTION
**Pull request by svenvonwerner**

### CODE REVIEW

**Review by lalueders — APPROVED** [review]
Good job

File: index.html [hunk 1]
```diff
@@ -11,11 +11,15 @@
...
-        </nav>
-        <nav>
+
             <a href="#biography">Biography</a>
         </nav>
     </header>
+    <figure>
+        <img src="https://source.unsplash.com/random/600x300" alt="An image of Sven von Werner">
+        <figcaption>A "T-Rex" on display.</figcaption>
+
```
Comment (badabam) [inline index.html:21]: Please remove this extra empty line.

File: index.html [hunk 2]
```diff
@@ -11,11 +11,15 @@
...
         <nav>
             <a href="#contact">Contact</a>
-        </nav>
-        <nav>
+
             <a href="#biography">Biography</a>
         </nav>
     </header>
+    <figure>
+        <img src="https://source.unsplash.com/random/600x300" alt="An image of Sven von Werner">
```
Comment (badabam) [inline index.html:19]: `"An image of ..."` is a duplication of content as the screen reader already reads out that this is an image. Please go with just `Sven von Werner` or in this case just an empty alt attribute: `alt=""`. It's your personal website and people know already your name from the main heading.

**Review by badabam — CHANGES_REQUESTED** [review]
Nice job overall! I found a thing for you to improve. Please change your code in VSCode, then add, commit and push so this pull request gets updated.

**Pull request closed without merging**